src/include/connection.h

```diff
@@ -268,6 +268,19 @@ struct __wt_connection_impl {
 	uint64_t  ckpt_time_recent;	/* Checkpoint time recent/total */
 	uint64_t  ckpt_time_total;
 
+	/* Checkpoint stats and verbosity timers */
+	struct timespec ckpt_start_time;/* Checkpoint start/stop time */
+	struct timespec ckpt_stop_time;
+					/* Checkpoint verbosity start time */
+	struct timespec ckpt_verb_start_time;
```

Comment:
ckpt_start_time vs ckpt_verb_start_time, we can rename variables in such a way that I can tell which one includes the scrub time and which one doesn't.